### PR TITLE
[Snyk] Fix for 1 vulnerabilities

### PR DESCRIPTION
**Pull request by Muhammad-Altabba**
<p>This PR was automatically created by Snyk using the credentials of a real user.</p><br /><h3>Snyk has created this PR to fix one or more vulnerable packages in the `yarn` dependencies of this project.</h3>

#### Changes included in this PR

- Changes to the following files to upgrade the vulnerable dependencies to a fixed version:
    - docs/package.json
    - docs/yarn.lock

#### Note for [zero-installs](https://yarnpkg.com/features/zero-installs) users

If you are using the Yarn feature [zero-installs](https://yarnpkg.com/features/zero-installs) that was introduced in Yarn V2, note that this PR does not update the `.yarn/cache/` directory meaning this code cannot be pulled and immediately developed on as one would expect for a zero-install project - you will need to run `yarn` to update the contents of the `./yarn/cache` directory. 
If you are not using zero-install you can ignore this as your flow should likely be unchanged.

##### With an upgrade:
Severity                   | Priority Score (*)                   | Issue                   | Breaking Change                   | Exploit Maturity
:-------------------------:|-------------------------|:-------------------------|:-------------------------|:-------------------------
medium severity  |  **658/1000**  <br/> **Why?** Proof of Concept exploit, Recently disclosed, Has a fix available, CVSS 5.3  | Regular Expression Denial of Service (ReDoS) <br/>[SNYK-JS-AXIOS-6124857](https://snyk.io/vuln/SNYK-JS-AXIOS-6124857) |  No  | Proof of Concept 

(*) Note that the real score may have changed since the PR was raised.

Check the changes in this PR to ensure they won't cause issues with your project.

------------

**Note:** *You are seeing this because you or someone else with access to this repository has authorized Snyk to open fix PRs.*

For more information:  
🧐 [View latest project report](https://app.snyk.io/org/muhammad-altabba/project/97fdd18d-9bab-4c87-87eb-da3caee72033?utm_source&#x3D;github&amp;utm_medium&#x3D;referral&amp;page&#x3D;fix-pr)

🛠 [Adjust project settings](https://app.snyk.io/org/muhammad-altabba/project/97fdd18d-9bab-4c87-87eb-da3caee72033?utm_source&#x3D;github&amp;utm_medium&#x3D;referral&amp;page&#x3D;fix-pr/settings)

📚 [Read more about Snyk's upgrade and patch logic](https://support.snyk.io/hc/en-us/articles/360003891078-Snyk-patches-to-fix-vulnerabilities)

[//]: # (snyk:metadata:{"prId":"00125ad7-a461-4c7e-93ff-695e78df17ea","prPublicId":"00125ad7-a461-4c7e-93ff-695e78df17ea","dependencies":[{"name":"@docusaurus/core","from":"3.0.0","to":"3.0.1"},{"name":"@docusaurus/theme-live-codeblock","from":"3.0.0","to":"3.0.1"}],"packageManager":"yarn","projectPublicId":"97fdd18d-9bab-4c87-87eb-da3caee72033","projectUrl":"https://app.snyk.io/org/muhammad-altabba/project/97fdd18d-9bab-4c87-87eb-da3caee72033?utm_source=github&utm_medium=referral&page=fix-pr","type":"auto","patch":[],"vulns":["SNYK-JS-AXIOS-6124857"],"upgrade":["SNYK-JS-AXIOS-6124857"],"isBreakingChange":false,"env":"prod","prType":"fix","templateVariants":["priorityScore"],"priorityScoreList":[658],"remediationStrategy":"vuln"})

---

**Learn how to fix vulnerabilities with free interactive lessons:**

 🦉 [Regular Expression Denial of Service (ReDoS)](https://learn.snyk.io/lesson/redos/?loc&#x3D;fix-pr)

### CODE REVIEW

**Review by avkos — APPROVED** [review]

File: docs/yarn.lock [hunk 1]
```diff
@@ -154,11 +154,24 @@
     "@babel/highlight" "^7.22.13"
     chalk "^2.4.2"
 
+"@babel/code-frame@^7.23.5":
```
Comment (jdevcs) [inline docs/yarn.lock:157]: was yarn lock generated by you?

Comment (Muhammad-Altabba) [reply docs/yarn.lock:157]: No, they were generated by Snyk.

Comment (jdevcs) [reply docs/yarn.lock:157]: I'll suggest lock files should be generated by our team internally.

**Comment by Muhammad-Altabba** [comment]
This dependencies update must be done along https://github.com/web3/web3.js/pull/6670 to pass the Docs Deploy. I am closing this MR while I included those updates in https://github.com/web3/web3.js/pull/6670

**Pull request closed without merging**